processor/src/lib.rs

```diff
@@ -881,6 +887,22 @@ impl<'a> ProcessState<'a> {
             ProcessState::Fast(state) => state.processor.memory.get_memory_state(ctx),
         }
     }
+
+    //
+    // HELPERS
+    // --------------------------------------------------------------------------------------------
+
+    pub(crate) fn cached_external_procedure(
```

Comment:
Better name?

Reply:
I initially misunderstood and moved the functions to `ProcessState`, but applied the correct suggestion in the latest commit.